www/src/components/email-capture-form.js

```diff
@@ -3,7 +3,6 @@ import { rhythm, options } from "../utils/typography"
 import presets, { colors } from "../utils/presets"
```

Comment:
While we're at this, could you remove the other unused imports here? 🙏 
See https://github.com/gatsbyjs/gatsby/pull/9045/files#diff-0b944fc95fef100bc9dd4aa6fbcb4647